.github/workflows/ci.yml

```diff
@@ -103,9 +103,16 @@ jobs:
         if: matrix.config.os == 'windows-latest'
         run: meson compile win-installer -C build
 
-      - name: Upload artifacts
+      - name: Upload artifacts - win_installer
         uses: actions/upload-artifact@v2
         if: matrix.config.os == 'windows-latest'
         with:
-          name: ${{ matrix.config.name }}
+          name: ${{ matrix.config.name }} - installer
           path: build/Aegisub-*.exe
+      
+      - name: Upload artifacts - portable.zip
+        uses: actions/upload-artifact@v2
+        if: matrix.config.os == 'windows-latest'
+        with:
+          name: ${{ matrix.config.name }} - portable
+          path: build/aegisub-portable-64.zip
```

Comment:
Is this actually going to work? I don't see a step calling `ninja win-portable`, so I would assume it just fails to find the artifact. It should be easy to check the artifacts from the PR and verify at least.